tests/ui/specs/Scroller/WithEditableSelectItemByPress/ScrollerWithEditableSelectItemByPress-specs.js

```diff
@@ -116,6 +116,137 @@ describe('Editable Scroller', function () {
+		await ScrollerPage.spotlightRight();
+		await ScrollerPage.spotlightRight();
+		await ScrollerPage.spotlightSelect();
+
+		// Verify: On the right side of Item 0 is Item 3.
+		await ScrollerPage.spotlightRight();
+		await expectFocusedItem(3);
+
+		// 5-way Left 3 times to check first item.
+		await ScrollerPage.moveSpotlight(2, 'spotlightLeft');
+		// Verify: On the left side of Item 0 is Item 2.
+		await expectFocusedItem(2);
+		await ScrollerPage.spotlightLeft();
+		// Verify: On the left side of Item 2 is Item 1.
+		await expectFocusedItem(1);
+		// Verify: First item is Item 1.
+		await ScrollerPage.spotlightLeft();
+		await ScrollerPage.spotlightDown();
+		await expectFocusedItem(1);
+
+		// Switch position of Item 1 and Item 2.
+		await ScrollerPage.spotlightSelect();
+		await ScrollerPage.spotlightRight();
+		await ScrollerPage.spotlightSelect();
+		await expectFocusedItem(1);
+		// Verify: On the right side of Item 1 is Item 0.
+		await ScrollerPage.spotlightRight();
+		await expectFocusedItem(0);
+		// Verify: Positions of Item 1 and Item 2 are switched and Item 2 is in the first position.
+		await ScrollerPage.moveSpotlight(2, 'spotlightLeft');
+		await expectFocusedItem(2);
+		await ScrollerPage.moveSpotlight(2, 'spotlightLeft');
+		await ScrollerPage.spotlightDown();
+		await expectFocusedItem(2);
+		await ScrollerPage.spotlightRight();
+		await expectFocusedItem(1);
+	});
+
+	it('should change item position with editableCentered', async function () {
+		// Set data size 1.
+		await ScrollerPage.inputFieldNumItems.moveTo();
+		await ScrollerPage.spotlightSelect();
+		await ScrollerPage.backSpace();
+		await ScrollerPage.backSpace();
+		await ScrollerPage.backSpace();
+		await ScrollerPage.numPad(1);
+
+		await ScrollerPage.spotlightDown();
+		await expectFocusedItem(0);
+		await ScrollerPage.spotlightSelect();
+
+		// Verify: Distance on the left and right side of the Item 0.
+		const itemLeftDistanceCentered = (await ScrollerPage.getActiveElementRect()).left;
+		const itemRightDistanceCentered = (await browser.getWindowSize()).width - (await ScrollerPage.getActiveElementRect()).right;
+
+		const expectedResult = Math.floor(itemLeftDistanceCentered / 100) * 100;
+		const result = Math.floor(itemRightDistanceCentered / 100) * 100;
+		await expect(expectedResult).to.be.equal(result);
+
+		// Set Editable Center to false.
+		await ScrollerPage.spotlightUp();
+		await ScrollerPage.spotlightUp();
+		await ScrollerPage.spotlightSelect();
+		await ScrollerPage.spotlightDown();
+
+		// Verify: Item 0 is not centered.
+		const itemLeftDistanceNotCentered = (await ScrollerPage.getActiveElementRect()).left;
+		await expect(itemLeftDistanceCentered).not.to.be.equal(itemLeftDistanceNotCentered);
+	});
+
+	it('should scroll through list', async function () {
+		// Move the spotlight to the last item that is not visible.
+		await ScrollerPage.spotlightDown();
+		await ScrollerPage.spotlightDown();
+		await ScrollerPage.moveSpotlight(9);
```

Comment:
to see in which direction we scroll
```suggestion
		await ScrollerPage.moveSpotlight(9, 'spotlightRight');
```